src/js/components/PodDetailInstancesTab.js

```diff
@@ -0,0 +1,34 @@
+import mixin from 'reactjs-mixin';
+import React from 'react';
+import {StoreMixin} from 'mesosphere-shared-reactjs';
+
+import Pod from '../structs/Pod';
+import PodView from './PodView';
+
+class PodDetailInstancesTab extends mixin(StoreMixin) {
+
+  constructor() {
+    super(...arguments);
+
+    this.store_listeners = [
+      {name: 'state', events: ['success']}
+    ];
+  }
+
+  render() {
+    return (
+      <PodView pod={this.props.pod} inverseStyle={true}
```

Comment:
Each prop on its own line please. So tagName and the first prop can be on the same line.